app/views/proposals/_encouragement_text.html.haml

```diff
@@ -2,7 +2,7 @@
   - if @program.event_types.any?
     You can submit proposals for
     = "#{event_types(@conference)}."
-  - if @program.tracks.any?
+  - if @program.tracks.confirmed.cfp_active.first
```

Comment:
Why not `@program.tracks.confirmed.cfp_active.any?`?

Reply:
@differentreality Yes, its better!